pkg/storkctl/migration.go

```diff
@@ -629,34 +660,30 @@ func ValidateMigration(migr *storkv1.Migration) error {
 	return nil
 }
 
-func GetMigrationScheduleCRNamespace(namespace string, config *rest.Config, ioStreams genericclioptions.IOStreams) string {
+func GetMigrationScheduleCRNamespace(namespace string) string {
 	var migrationScheduleNamespace string
-
-	dynamicClient, err := k8sdynamic.NewForConfig(config)
-	if err != nil {
-		util.CheckErr(err)
-		return migrationScheduleNamespace
-	}
-
+	dynamicClient := dynamicops.Instance()
 	// Checking in following resources for migrationschedule namespace annotation
 	resources := []struct {
-		kind      string
-		gvr       schema.GroupVersionResource
-		namespace string
+		kind       string
+		apiVersion string
 	}{
-		{"PersistentVolumeClaim", schema.GroupVersionResource{Group: "", Version: "v1", Resource: "persistentvolumeclaims"}, namespace},
-		{"StatefulSet", schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "statefulsets"}, namespace},
-		{"Deployment", schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}, namespace},
-		{"Service", schema.GroupVersionResource{Group: "", Version: "v1", Resource: "services"}, namespace},
-		{"ConfigMap", schema.GroupVersionResource{Group: "", Version: "v1", Resource: "configmaps"}, namespace},
+		{"PersistentVolumeClaim", "v1"},
+		{"StatefulSet", "apps/v1"},
+		{"Deployment", "apps/v1"},
+		{"Service", "v1"},
+		{"ConfigMap", "v1"},
```

Comment:
Let's add more resource types to figure out the migraitonschedule namespace.

Reply:
Makes sense. Will add as discussed